### ISSUE TITLE
Geting so many errors using this on my server

### ISSUE THREAD

**Issue by KyberNetworks** [MattJeanes/TARDIS#638]
Getting loads of errors on my server like 
```
[TARDIS] lua/entities/gmod_tardis/modules/sh_doors.lua:230: attempt to index field 'metadata' (a nil value)
1. v - lua/entities/gmod_tardis/modules/sh_doors.lua:230
2. CallHook - lua/entities/gmod_tardis/shared.lua:54
3. unknown - lua/entities/gmod_tardis/modules/sh_doors.lua:226
4. func - lua/entities/gmod_tardis/modules/libraries/sh_message.lua:34
5. unknown - lua/includes/extensions/net.lua:33

```

and others like

```
[TARDIS] lua/entities/gmod_tardis/modules/sh_lock.lua:85: attempt to index field 'metadata' (a nil value)
    1. unknown - lua/entities/gmod_tardis/modules/sh_lock.lua:85
        2. func - lua/entities/gmod_tardis/modules/libraries/sh_message.lua:34
            3. unknown - lua/includes/extensions/net.lua:33
```

**Comment by KyberNetworks** [comment]
[TARDIS] lua/tardis/sh_screen_ui.lua:753: attempt to call method 'CallHook' (a nil value)
  1. unknown - lua/tardis/sh_screen_ui.lua:753 (x5214)

**Comment by parar020100** [comment]
Hi
Sorry for taking long to answer
This is related to metadata indexing, a rather annoying bug we haven't yet been able to fix
You can track the progress of it being fixed here: #334
